.nvmrc

```diff
@@ -1 +1 @@
-v18
+v22
```

Comment:
Maybe we should use the same minor version that's in the Dockerfile?  I don't think there's much benefit to leaving it unpinned.